services/stackexchange/stackexchange-reputation.service.js

```diff
@@ -37,46 +37,44 @@ module.exports = class StackExchangeReputation extends BaseJsonService {
   static get examples() {
     return [
       {
-        title: 'StackOverflow',
+        title: 'Stack Exchange reputation',
         namedParams: { stackexchangesite: 'stackoverflow', query: '123' },
         staticExample: this.render({
-          label: 'stackoverflow reputation',
+          stackexchangesite: 'stackoverflow',
           numvalue: 10,
-          color: floorCountColor(1000, 10000, 20000),
         }),
         keywords: ['stackexchange', 'stackoverflow'],
       },
     ]
   }
 
-  static render({ label, numValue, color }) {
+  static render({ stackexchangesite, numValue }) {
+    const label = `${stackexchangesite} reputation`
+
     return {
       label,
       message: metric(numValue),
-      color,
+      color: floorCountColor(numValue),
```

Comment:
To keep the intended color scale, this should be `floorCountColor(numValue, 1000, 10000, 20000)`.

Reply:
Makes sense thanks for catching that